lib/units/ios-device/plugins/devicelog.js

```diff
@@ -82,7 +87,7 @@ export default syrup.serial()
                                 realGroup.group
                                 , wireutil.envelope(new wire.DeviceLogcatEntryMessage(
                                     options.serial
-                                    , new Date().getTime() / 1000
+                                    , Date.parse(timestamp) / 1000
```

Comment:
Default value of timestamp in case no timestamp could be found from the log line entry is not parseable
<img width="322" alt="image" src="https://github.com/user-attachments/assets/cd094ee9-629a-49e7-926c-3df618fb57e9" />